addons/project/models/ir_ui_menu.py

```diff
@@ -11,4 +11,6 @@ def _load_menus_blacklist(self):
         res = super()._load_menus_blacklist()
         if self.env.user.has_group('project.group_project_stages'):
             res.append(self.env.ref('project.menu_projects').id)
+        if self.env.user.has_group('project.group_project_user') and not self.env.user.has_group('project.group_project_manager'):
```

Comment:
I think it is equivalent to:
```suggestion
        if self.user_has_groups('project.group_project_user,!project.group_project_manager'):
```

Reply:
Done